apps/app/pages/admin/event/[id].tsx

```diff
@@ -17,14 +17,12 @@ function EventPage() {
   const { id: event_id } = router.query;
 
   const { data: event, isLoading } = useEvent(event_id as string);
-
-  const [mentors, setMentors] = useState<any[]>([]);
   const [enrolledNinjas, setEnrolledNinjas] = useState([]);
   const [availableMentors, setAvailableMentors] = useState<any[]>([]);
   const [unavailableMentors, setUnavailableMentors] = useState<any[]>([]);
   const [available, setAvailable] = useState<boolean>(true);
   useEffect(() => {
-    getAvailableMentors(event_id as string)
+    getMentors(event_id as string)
       .then((response: any) => {
         setAvailableMentors(
           response.data.filter((mentor: any) => mentor.is_available)
```

Comment:
This filtering would make the page not capable of showing the unavailable mentors.